server/routes/journeys/base/controller.ts

```diff
@@ -41,4 +41,11 @@ export class BaseJourneyController {
       text: refData.description,
       checked: Array.isArray(value) ? value.includes(refData.code) : refData.code === value,
     }))
+
+  deleteJourneyDataAndGoBackToCsipRecordPage = async (req: Request, res: Response) => {
+    const { recordUuid } = req.journeyData.csipRecord!
+    // @ts-expect-error delete non-optional req.journeyData to free up redis memory
+    delete req.journeyData
```

Comment:
Does this clear in redis as well or just locally?

Reply:
persistence of req.journeyData is managed by the setupJourneyDataMiddleware
it deletes the data from redis if req.journeyData is undefined (deleted)